content/en/docs/reference/node/kubelet-files.md

```diff
@@ -0,0 +1,114 @@
+[drop-in configuration files](/docs/tasks/administer-cluster/kubelet-config-file/#kubelet-conf-d)
+to enhance configuration.
+
+#### Certificates
+
+Certificates are typically located at `/var/lib/kubelet/pki`,
+but can be configured using the `--cert-dir` kubelet command line argument.
+Names of certificate files are also configurable.
+
+#### Systemd unit settings
+
+When kubelet is running as a systemd unit, some kubelet configuration may be declared
+in systemd unit settings file. Typically it includes:
+
+- command line arguments to [run kubelet](/docs/reference/command-line-tools-reference/kubelet/)
+- environment variables, used by kubelet or [configuring golang runtime](https://pkg.go.dev/runtime#hdr-Environment_Variables)
+
+### State
+
+#### Manifests
+
+Manifests for static pods are typically located in `/etc/kubernetes/manifests`.
+Location can be configured using the `staticPodPath` kubelet configuration option.
+
+#### Checkpoint files
+
+All resource managers keep the mapping of Pods to allocated resources in state files.
+State files located in the kubelet's root directory,
+typically `/var/lib/kubelet`. Kubelet root directory can be configured
+using the kubelet command line argument `--root-dir`.
+
+Names of files:
+
+- `memory_manager_state` for the Memory Manager
+- `cpu_manager_state` for the CPU Manager
+- `kubelet_internal_checkpoint` for Device Manager
+- `dra_manager_state` for DRA
+
+#### Pod Status checkpoint
+
+{{< feature-state feature_gate_name="InPlacePodVerticalScaling" >}}
+
+The pod status is being checkpointed to implement the Pod resize feature.
+The file name is `pod_status_manager_state` in kubelet root directory
+(`/var/lib/kubelet` or configure using `--root-dir`).
+
+### Sockets
+
+Kubelet opens a few sockets as well as reading some other sockets.
+
+#### Container runtime
+
+Kubelet communicates with the container runtime using socket configured via the
+configuration parameters:
+
+- `containerRuntimeEndpoint` for runtime operations
+- `imageServiceEndpoint` for image management operations
+
+The actual values of those endpoints depend on the container runtime being used.
+
+#### Device plugins
+
+Kubelet exposes socket `/var/lib/kubelet/device-plugins/kubelet.sock` for
+various [Device Plugins to register](/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#device-plugin-implementation).
+
+When a device plugin registers itself, it provides its socket path for the kubelet to connect.
+
+The device plugin socket should be in the directory `/var/lib/kubelet/device-plugins`.
+
+#### Pod Resources API
+
+[Pod Resources API](/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#monitoring-device-plugin-resources)
+will be exposed at the address `/var/lib/kubelet/pod-resources`.
+
+#### DRA and CSI plugins
+
+DRA and CSI plugins are located in `plugins_registry` in kubelet's root directory
+(typically `/var/lib/kubelet/plugins_registry`).
+
+### Utility
```

Comment:
```suggestion
### Locking

{{< feature-state state="alpha" for_k8s_version="v1.2" >}}

```

This might be older, but I think it comes from the 1.2 release.